uaclient/apt.py

```diff
@@ -66,12 +66,25 @@ def add_auth_apt_repo(repo_filename, repo_url, credentials, keyring_file=None,
     if not valid_apt_credentials(repo_url, username, password):
         raise InvalidAPTCredentialsError(
             'Invalid APT credentials provided for %s' % repo_url)
+    policy, _err = util.subp(['apt-cache', 'policy'])
+    updates_enabled = bool('%s-updates/' in policy)
+
     logging.info('Enabling authenticated repo: %s', repo_url)
     content = ''
-    for pocket in pockets:
-        content += ('deb {url}/ubuntu {series} {pocket}\n'
-                    '# deb-src {url}/ubuntu {series} {pocket}\n'.format(
-                        url=repo_url, series=series, pocket=pocket))
+    if not suites:
+        logging.debug('No apt source suite provided, assuming "%s".', series)
+        suites = [series]  # The default suite should be the series name
```

Comment:
I debate whether suites should always be required, as we wouldn't ever have a xenial for esm

Reply:
Sure let's complain loudly. I'm seeing  suites for esm, cc and fips defined by ua-contracts, so I'm good with that